Readme.md

```diff
@@ -54,41 +54,142 @@ We prepared some defaults for testnet and mainnet, however to run it locally, or
 
-> ### Warning :warning
+<details open>
+<summary><small>./config/default.json5</small></summary>
+
+```json5
+// config/default.json5
+
+// TODO: add descriptions in comments
+// This file should not be aimed at any specific environment, but rather contain configuration defaults that are not likely to cause issues if left undefined in an override
+{
+  /*
+    Server 
+  */
+  app: {
+    url: 'http://127.0.0.1', // URL where the relay server will be deployed, it could be localhost or the IP of the host machine.
+    port: 8090, // port where the relay server will be hosted.
+    devMode: false, // indicates to the server if we are in development mode or not.
+    customReplenish: false, // set if the server uses a custom replenish function or not.
+
+    logLevel: 4 /* The log level for the relay server. Available levels:
+      {
+        TRACE: 0;
+        DEBUG: 1;
+        INFO: 2;
+        WARN: 3;
+        ERROR: 4;
+        SILENT: 5;
+      }
+    */,
+    workdir: '.', // path to the folder where the server will store the database and all its data.
+    readyTimeout: 30000,
+    checkInterval: 10000,
+    disableSponsoredTx: false,
+    feePercentage: 0 /* allows revenue sharing feature and sets the fee value (%) that the worker will take from all transactions.
+    - the fee will be added to the estimated gas and required in the transaction amount.
+    - the percentage is represented as a fraction (1 = 100%) string to allow for very low or high percentages
+    - the minus sign is omitted if used
+    - fractions exceeding the number of decimals of that of the native currency will be rounded up
+   */,
+    sponsoredDestinations: [],
+    requestMinValidSeconds: 43200,
+  },
+  /*
+    Blockchain node
+  */
+  blockchain: {
+    rskNodeUrl: 'http://127.0.0.1:4444', //  RSK node endpoint URL, where the RSK node is located.
+    gasPriceFactor: 1,
+    alertedBlockDelay: 0,
+    minAlertedDelayMS: 0,
+    maxAlertedDelayMS: 0,
+    registrationBlockRate: 0,
+    workerMinBalance: 0.001e18, // 0.001 RBTC
+    workerTargetBalance: 0.003e18, // 0.003 RBTC
+    managerMinBalance: 0.001e18, // 0.001 RBTC
+    managerMinStake: 1, // 1 wei
+    managerTargetBalance: 0.003e18, // 0.003 RBTC
+    minHubWithdrawalBalance: 0.001e18, // 0.001 RBTC
+    refreshStateTimeoutBlocks: 5,
+    pendingTransactionTimeoutBlocks: 30, // around 5 minutes with 10 seconds block times.
+    successfulRoundsForReady: 3, // successful mined blocks to become ready after exception.
+    confirmationsNeeded: 12,
+    retryGasPriceFactor: 1.2, // gas price factor used to calculate the gas on the server, you can leave it as 1.
+    defaultGasLimit: 500000,
+    maxGasPrice: 100000000000,
+    estimateGasFactor: 1.2,
+    versionRegistryDelayPeriod: 0,
+  },
+  /*
+    Relay contracts addresses
+  */
+  contracts: {
+    relayHubAddress: '', // relay hub contract address, you can retrieve this from the contract summary.
```

Comment:
The readme doesn't reflect the latest changes on `default.json5` since we're using double quotes now

Reply:
resolved with the above